Bottom: 6cdce76964cf173251715d6e7e244c69a2156144
Top:    37543103c35e376f8b1297c0607bc4beb8d13ff3
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-03 13:17:22 +0000

Fix GDK build


---

diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index b680c3f..0c840a3 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -90,11 +90,11 @@ ghid_set_layer (const char *name, int group, int empty)
     case SL_INVISIBLE:
       return /* pinout ? 0 : */ PCB->InvisibleObjectsOn;
     case SL_MASK:
-      if (SL_MYSIDE (idx) /*&& !pinout */ )
+      if (SL_MYSIDE (group) /*&& !pinout */ )
         return TEST_FLAG (SHOWMASKFLAG, PCB);
       return 0;
     case SL_SILK:
-      if (SL_MYSIDE (idx) /*|| pinout */ )
+      if (SL_MYSIDE (group) /*|| pinout */ )
         return PCB->ElementOn;
       return 0;
     case SL_ASSY:
